astropy/convolution/convolve.py

```diff
@@ -304,50 +306,48 @@ def convolve(array, kernel, boundary='fill', fill_value=0.,
                 array_to_convolve[pad_width[0]:array_shape[0]+pad_width[0]] = array_internal
             elif array_internal.ndim == 2:
                 array_to_convolve[pad_width[0]:array_shape[0]+pad_width[0],
-                             pad_width[1]:array_shape[1]+pad_width[1]] = array_internal
-            elif array_internal.ndim == 3:
+                                  pad_width[1]:array_shape[1]+pad_width[1]] = array_internal
+            else:
```

Comment:
Was this change discussed?

Reply:
ndim cannot be >3, line 244 already throws a `NotImplementedError`. But on the other hand I cannot remember why I changed it, it shouldn't matter.

Reply:
Okie dokie then. Thanks for the clarification!